metrics/cadvisor.go

```diff
@@ -170,7 +170,7 @@ func (cAdvisor *CAdvisor) GetMetrics2() (*MetricsBatch, error) {
 
 		return cAdvisor.withBackoff(func() error {
 			nodeMetricsBatch, err := ReadPrometheusMetrics(
-				"https://"+cAdvisor.getNodeKubeletAddress(node)+"/metrics/cadvisor",
```

Comment:
@YasserRabee will this work without the kubelet port commit?

Reply:
cadvisor metrics will work on http port. We are already doing that.

Reply:
here it is using https
https://github.com/MagalixCorp/magalix-agent/pull/7/files#diff-a4f6a21f1d4b814bb22ae5c16546906dR272

but in case this PR is rebased to the newest dev branch this will NOT work because `getNodeKubeletAddress` method is returning the url WITHOUT the schema
https://github.com/MagalixCorp/magalix-agent/blob/dev/metrics/metrics.go#L236